DigitalOffice/Tests/AuthenticationServiceTests/Commands/UserLoginCommandTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using FluentValidation;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using MassTransit;
+using Moq;
+using NUnit.Framework;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.AuthenticationService.Token.Interfaces;
+using LT.DigitalOffice.AuthenticationService.Commands;
+using LT.DigitalOffice.AuthenticationService.Models;
+using LT.DigitalOffice.AuthenticationService.Token;
+using LT.DigitalOffice.AuthenticationService.Validators;
+using LT.DigitalOffice.AuthenticationService.Commands.Interfaces;
+using LT.DigitalOffice.AuthenticationServiceUnitTests.UnitTestLibrary;
+using LT.DigitalOffice.AuthenticationService.Broker.Senders.Interface;
+using LT.DigitalOffice.AuthenticationServiceTests.Broker.Senders;
+using LT.DigitalOffice.AuthenticationService.Broker.Senders;
+
+namespace LT.DigitalOffice.AuthenticationServiceUnitTests.Commands
+{
+    public class UserLoginCommandTests
+    {
+        private Mock<Response<IOperationResult<IUserCredentialsResponse>>> responseBrokerMock;
+        private Mock<IRequestClient<IUserCredentialsRequest>> requestBrokerMock;
+        private IValidator<UserLoginCredentialsRequest> validator;
+        private UserLoginCredentialsRequest newUserCredentials;
+        private UserCredentialsResponse brokerResponse;
+        private ISender<string, object> brokerSender;
+        private IUserLoginCommand command;
+        private NewToken newToken;
+        private string userEmail;
```

Comment:
Too many mocks. You need to mock only NewToken, ISender, IValidator

Reply:
Fixed